synfig-studio/src/gui/widgets/widget_coloredit.cpp

```diff
@@ -427,6 +427,23 @@ Widget_ColorEdit::Widget_ColorEdit():
 		//I use Gtk::ColorSelection widget here.
 		hvsColorWidget = manage(new Gtk::ColorSelection());
 		setHVSColor(get_value());
+
+		//navigating to inner implementation color wheel widget
+		std::vector<Widget*> internal_child = hvsColorWidget->get_children();
+		Gtk::Box* box_cast = static_cast<Gtk::Box*>( internal_child[0] );
+		std::vector<Widget*> internal_child_2 = box_cast->get_children();
+		Gtk::Box* box_cast_2 = static_cast<Gtk::Box*>( internal_child_2[0] );
+		std::vector<Widget*> internal_child_3 = box_cast_2->get_children(); //internal_child_3[0] is the color wheel widget
+
+		internal_child_3[0]->signal_button_press_event().connect([&](GdkEventButton *ev){ colorHVSChanged = false;escape_cancel=false; return false;},false);
```

Comment:
Ok, now it should be possible to remove this part too:
```
internal_child_3[0]->signal_button_press_event().connect([&](GdkEventButton *ev){ colorHVSChanged = false;escape_cancel=false; return false;},false);
internal_child_3[0]->signal_button_release_event().connect([&](GdkEventButton *ev){ if((hvsColorWidget->is_adjusting()))wheel_released=true;return false;},false);
```

When you press the mouse down - it emits `signal_color_changed()` event with `is_adjusting() = true` flag.
When you release the mouse button - it emits `signal_color_changed()` event with `is_adjusting() = false` flag.

Reply:
removed the button press event, but the release event is needed to diffrentiate between color change from wheel vs color change from elsewhere, as also color change not from color wheel emits `signal_color_changed()` event with `is_adjusting() = false` flag.

Reply:
> as also color change not from color wheel emits `signal_color_changed()` event with `is_adjusting() = false` flag.

Yes. In that case `setHVSColor()` -> `signal_color_changed()` is called. 
To distinguish manual color adjustment from the color wheel, the `colorHVSChanged` flag was previously used.
You can add your own flag and use it to test this condition. Next step will be merging this both flags (this will simplify the code).